script/check_same_primer_view_components_version_everywhere

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+echo "Checking for same primer view components version in Gemfile.lock and frontend/package.json"
+
+versions=$(grep --color=never primer.view.components frontend/package.json Gemfile.lock \
+          | grep --extended-regexp --only-matching '\d+\.\d+\.\d+' \
+          | uniq)
+version_count=$(echo "$versions" | wc -l)
```

Comment:
Admittedly, grepping the `package-lock.json` and `Gemfile.lock` is going to be significantly faster.

However, it might be more robust to use the published `bundle` and `npm` commands to the same thing.

```suggestion
function bundle_version {
  bundle info --version openproject-primer_view_components
}

function npm_version {
  npm ls --omit=dev --json --package-lock-only @primer/view-components @openproject/primer-view-components | 
    jq -r '[.dependencies[].version] | unique[0]'
}

versions=$(printf "%s\n%s" $(bundle_version) $(npm_version))
versions_count=$(echo "$versions" | wc -l)
```

Reply:
unsurprisingly grepping is definitely faster:

```
./test.sh  0.84s user 0.13s system 98% cpu 0.977 total # (my suggestion)
./test2.sh  0.01s user 0.02s system 103% cpu 0.021 total # (your script)
```

Reply:
grepping is good enough. It's to prevent small mistakes.